tools/create_data.py

```diff
@@ -171,53 +173,75 @@ def waymo_data_prep(root_path,
                     version,
                     out_dir,
                     workers,
-                    max_sweeps=5):
-    """Prepare the info file for waymo dataset.
+                    max_sweeps=10,
+                    only_gt_database=False,
+                    skip_image_and_lidar=False,
```

Comment:
change to `save_image_and_lidar` and could consider naming it as `save_sensor_data`